bib/progress.bib

```diff
@@ -61,7 +61,8 @@ @ARTICLE{cohn2020tsbp
   pages={6694--6701},
   publisher={IEEE},
   video={https://www.youtube.com/watch?v=fAFwI7ll3bg},
-  url={https://ieeexplore.ieee.org/document/9166624}
+  url={https://ieeexplore.ieee.org/document/9166624},
+  webpage={projects/tsbp}
```

Comment:
Can you rerun the script `scripts/gen_pubs`? The server doesn't run it automatically right now, so this change won't show up.